src/test/test-copy.c

```diff
@@ -50,6 +50,85 @@ TEST(copy_file) {
         unlink(fn_copy);
 }
 
+TEST(copy_tree_replace_file) {
+        _cleanup_free_ char *buf = NULL;
+        char *src, *dst;
```

Comment:
Missing _cleanup_free_ on these variables (and they should be merged into the above declaration